docs/modules/ROOT/pages/explanations/decisions/local-objectstorage.adoc

```diff
@@ -0,0 +1,100 @@
+= VSHN Managed Object Storage
+
+== Problem
+
+Having access to S3 compatible storage is required for VSHN managed AppCat services, as well as the APPUiO Managed cluster itself for backups.
+Further it is often used by other applications such as logging and monitoring systems, or directly by the customer.
+
+Most platforms on which we run APPUiO Managed already provides some kind of S3 compatible storage and whenever possible AppCat directly interfaces with it to provide self-service object buckets.
```

Comment:
This is partly true, we actually only support S3 storage on clusters running on exoscale or cloudscale.ch.
This sentence obviously is correct at the moment. But might not be true anymore in the future, if a lot of on-premise clusters are set up even if they have some kind of S3 compatible storage, because we lack the necessary provider for that on-premise S3 implementation.